feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -86,18 +86,8 @@ func TestBasic(t *testing.T) {
 			check.Equal(isisRoot.Global().LevelCapability().State(), oc.Isis_LevelType_LEVEL_2),
 			check.Equal(port1ISIS.Enabled().State(), true),
 			check.Equal(port1ISIS.CircuitType().State(), oc.Isis_CircuitType_POINT_TO_POINT),
-		}
-
-		// if MissingIsisInterfaceAfiSafiEnable is set, ignore enable flag check for AFI, SAFI at global level
-		// and validate enable at interface level
-		if deviations.MissingIsisInterfaceAfiSafiEnable(ts.DUT) {
```

Comment:
This deviation is also used by Nokia in this test, I dont think we can remove this deviation check from the test itself.  Same for otg_test

Reply:
okay , i will revert this check if its failing for them.